src/6_count_occurences.js

```diff
@@ -1,3 +1,20 @@
-module.exports = function countOccurences() {
-  // TODO Implement me.
+function countOccurences(str) {
+  words = str.split(" ");
+  const obj = {};
+  const objFlip = {};
+
+  for (const word of words) {
+    !obj[word] ? obj[word] = 1 : obj[word] += 1;
```

Comment:
trenary side effect 
https://stackoverflow.com/questions/55349332/using-ternary-operator-just-for-the-side-effect-good-or-bad